DeusEx/Classes/DXRSwapItems.uc

```diff
@@ -1,9 +1,28 @@
 class DXRSwapItems extends DXRActorsBase;
 
+var config class<Actor> swap_actors[16];
+
+function CheckConfig()
+{
+    local int i;
+    if( config_version == 0 ) {
+        for(i=0; i < ArrayCount(swap_actors); i++) {
+            swap_actors[i] = None;
+        }
+        i=0;
+        swap_actors[i++] = class'Inventory';
+        swap_actors[i++] = class'Containers';
```

Comment:
now you could change your config to swap all ScriptedPawns or something crazy, maybe we need a flag to ensure it doesn't swap ScriptedPawns with bImportant==true